packages/lib-ts/src/svm.ts

```diff
@@ -0,0 +1,18 @@
+import { JSON } from 'json-as'
+
+import { SvmFindProgramAddressParams, SvmFindProgramAddressResult } from './types/SvmFindProgramAddress'
+
+export namespace svm {
+  @external('svm', '_findProgramAddress')
+  declare function _findProgramAddress(params: string): string
+
+  /**
+   * Calculates PDA address
+   * @param params - Seeds, program address
+   * @returns The PDA address as base58
+   */
+  export function findProgramAddress(params: SvmFindProgramAddressParams): SvmFindProgramAddressResult {
```

Comment:
I wouldn't receive the object in here, I think it's better to receive the parameters and create the seed and the SvmFindProgramAddressParams in here. It's easier.

We did it with the EvmEncode and EvmDecode and it's not the best UX specially because we have few arguments with no optional